dev/docker-compose-tss.yml

```diff
@@ -26,7 +26,7 @@ services:
       - -c
       - |
         sleep 10
-        ./stellar-disbursement-platform channel-accounts verify --delete-invalid-accounts
+        ./stellar-disbursement-platform channel-accounts delete --delete-all-accounts
```

Comment:
I think you merged develop into this branch. I would prefer if we do these merges separately from feature PRs as we merge them differently.